docs/customisations.md

```diff
@@ -0,0 +1,65 @@
+# Extending and customising the platform app
+
+## Section structure
+
+The target profile tab of the target page is broken up into sections. Each of these sections is encapsulated in one directory. To add a new one, try to follow the following layout guide:
+
+```
+target/
+ sections/
+   ...
+   <section-id>/
+     custom/
+       ... // it may help to put any other components you create here,
+           // to keep them separate from the template/expected ones below
+     Description.js
+     index.js
+     Section.js
```

Comment:
Body.js?

Reply:
Or is this pull request not up to date with <https://github.com/opentargets/platform/issues/1131#issuecomment-697681546>?

Reply:
Good catch, that I have to remove that part of the section.